src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -46,6 +46,10 @@ ClrUnwindEx(EXCEPTION_RECORD* pExceptionRecord,
                  UINT_PTR          TargetFrameSp);
 #endif // !TARGET_UNIX
 
+#ifdef HOST_UNIX
+VOID UnwindManagedExceptionPass2(PAL_SEHException& ex, CONTEXT* unwindStartContext);
+#endif // HOST_UNIX
```

Comment:
```suggestion
#if defined(TARGET_UNIX) && !defined(DACCESS_COMPILE)
VOID UnwindManagedExceptionPass2(PAL_SEHException& ex, CONTEXT* unwindStartContext);
#endif // TARGET_UNIX && !DACCESS_COMPILE
```
This would be the way to ifdef it in case if you wanted to stick with `TARGET_UNIX`. (You can keep HOST_UNIX here if you prefer.)